togetherly/lib/services/chore_service.dart

```diff
@@ -4,9 +4,8 @@ import 'package:togetherly/models/chore.dart';
 class ChoreService {
   static const String _choreTable = "chore";
 
-  Future<List<Chore>> getChores(int familyId) async {
-    var result = await Supabase.instance.client.from(_choreTable).select(
-        'id, title, description, date_due, points, status, shared');
+  Future<List<Chore>> getFamilyChores(int familyId) async {
```

Comment:
Do you think `getChoresByFamily` or `getChoresByFamilyId` would be a better name? I feel like `getFamilyChores` implies some connection to the Family model.